config/main.py

```diff
@@ -15,6 +15,7 @@
 from swsssdk import ConfigDBConnector
 from swsssdk import SonicV2Connector
 from minigraph import parse_device_desc_xml
+from click_default_group import DefaultGroup
```

Comment:
Is it necessary to import click_default_group to accomplish this? We don't use the functionality of DefaultGroup any longer, and I was planning to remove it entirely. Can instead inherit a Click class?

Reply:
DefaultGroup inherits from `click.Group`. Can you try simply inheriting this class and see if this works?

Reply:
We can merge this as-is and attempt to replace DefaultGroup with click.Group in a separate PR.